cmd/opdoc/opdoc.go

```diff
@@ -221,10 +246,11 @@ func opToMarkdown(out io.Writer, op *logic.OpSpec, groupDocWritten map[string]bo
 
 func opsToMarkdown(out io.Writer) (err error) {
 	out.Write([]byte("# Opcodes\n\nOps have a 'cost' of 1 unless otherwise specified.\n\n"))
-	opSpecs := logic.OpcodesByVersion(logic.LogicVersion)
+
 	written := make(map[string]bool)
-	for _, spec := range opSpecs {
-		err = opToMarkdown(out, &spec, written)
+	for _, name := range logic.OpNames {
+		specs := logic.SpecsByName(name)
```

Comment:
At some point it would be nice to use OpGroups to get these documented in a coherent order. Consider writing that, but _don't_ do it in the same PR. By keeping the same order for now, we can review changes to the generated docs more easily.